src/components/interface/MellowLpDepositForm/components/LPMellowVaultDepositWindow.tsx

```diff
@@ -146,21 +152,26 @@ const LPMellowVaultDepositWindow: React.FunctionComponent<LPMellowVaultDepositWi
           variant="h1"
           sx={{
             fontSize: '14px',
-            lineHeight: '1.4',
             color: '#C6C2DA',
             fontWeight: '700',
             marginLeft: '0px',
+            lineHeight: '120%',
           }}
         >
           ABOUT YOUR FUNDS
         </Typography>
         <Typography
           variant="body1"
-          sx={{ fontSize: '12px', color: '#9B97AD', marginTop: '10px', fontWeight: '400' }}
+          sx={{
+            fontSize: '12px',
+            color: '#9B97AD',
+            marginTop: '10px',
```

Comment:
```suggestion
            marginTop: (theme) => theme.spacing(2.5),
```

Reply:
fixed in a880d81b18fc358da6db21e427119ac7f03e85ef